common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -80,47 +81,53 @@
+                                HtmlUtils.HTML('<span class="sr control-text"></span>'),
+                            HtmlUtils.HTML('</span>'),
+                        HtmlUtils.HTML('</button>'),
+                        HtmlUtils.HTML('<button class="control toggle-transcript" aria-disabled="false">'),
+                            HtmlUtils.HTML('<span class="icon-fallback-img">'),
+                                HtmlUtils.HTML('<span class="icon fa fa-quote-left" aria-hidden="true"></span>'),
+                                HtmlUtils.HTML('<span class="sr control-text"></span>'),
+                            HtmlUtils.HTML('</span>'),
+                        HtmlUtils.HTML('</button>'),
+                        HtmlUtils.HTML('<div class="lang menu-container" role="application">'),
+                            HtmlUtils.HTML('<p class="sr instructions" id="lang-instructions"></p>'),
+                            HtmlUtils.HTML('<button class="control language-menu" aria-disabled="false"'),
+                                HtmlUtils.HTML('aria-describedby="lang-instructions" '),
+                                HtmlUtils.HTML('title="'),
                                     gettext('Open language menu'),
-                                '">',
-                                '<span class="icon-fallback-img">',
-                                    '<span class="icon fa fa-caret-left" aria-hidden="true"></span>',
-                                    '<span class="sr control-text"></span>',
-                                '</span>',
-                            '</button>',
-                        '</div>',
-                    '</div>'
-                ].join('');
-
-                var template = [
-                    '<div class="subtitles" role="region" id="transcript-' + this.state.id + '">',
-                        '<h3 id="transcript-label-' + this.state.id + '" class="transcript-title sr"></h3>',
-                        '<ol id="transcript-captions" class="subtitles-menu" lang="' + this.state.lang + '"></ol>',
-                    '</div>'
-                ].join('');
+                                HtmlUtils.HTML('">'),
+                                HtmlUtils.HTML('<span class="icon-fallback-img">'),
+                                    HtmlUtils.HTML('<span class="icon fa fa-caret-left" aria-hidden="true"></span>'),
+                                    HtmlUtils.HTML('<span class="sr control-text"></span>'),
+                                HtmlUtils.HTML('</span>'),
+                            HtmlUtils.HTML('</button>'),
+                        HtmlUtils.HTML('</div>'),
+                    HtmlUtils.HTML('</div>)')
+                );
+
+                var subtitlesHtml = HtmlUtils.interpolateHtml(
+                    HtmlUtils.joinHtml(
+                        HtmlUtils.HTML('<div class="subtitles" role="region" id="transcript-{courseId}">'),
```

Comment:
In my opinion, the original was easier to read and I'd follow the same pattern with a single `HTML()` call documented above.